File: Espd/typings/ContractingPartyTypeType.ts



import { CodeType } from './CodeType'

import assign from 'lodash.assign'

export class ContractingPartyTypeType { 
    public constructor(init?:Partial<ContractingPartyTypeType>) {
        assign(this, init)
    } 
    
    
    partyTypeCode?: CodeType
    partyType?: string
}

File: UBL/CommonAggregateComponents/ContractingPartyTypeType.cs
using Hilma.UBL.Attributes;
using Hilma.UBL.UnqualifiedDataTypes;

namespace Hilma.UBL.CommonAggregateComponents
{
  [Contract]
  public class ContractingPartyTypeType
  {
    /// <summary>
    /// 
    /// </summary>
    public CodeType PartyTypeCode { get; set; }

    public string PartyType { get; set; }

  }

}
